cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -259,6 +259,7 @@ spec:
 #      - name: "nvme01" # multiple osds can be created on high performance devices
 #        config:
 #          osdsPerDevice: "5"
+#      - name: "/dev/disk/by-id/ata-ST4000DM004-XXXX" # devices can be specified using full udev paths
```

Comment:
`fullPath` would need to be specified in the crd instead of `name`, correct?

Reply:
I was thinking about this: I think just using `name` is preferable as the name could be misleading if device names change. Also specifying both is redundant.

What do you think? Should `fullPath` be removed?

Reply:
Since `name` and `fullPath` should never need to be used together and we can also detect whether the value is a full path (with the `/dev/` prefix), then it does seem like `fullPath` can be removed.